modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,306 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_content-type: PROCEDURE
+[id="creating-automated-etcd-backups_{context}"]
+= Creating automated etcd backups
+
+The automated backup feature for etcd supports both recurring and single backups. Recurring backups are implemented as a cronjob that starts a single backup each time it triggers.
+
+You can set the method of backup retention to `RetentionNumber`, `RetentionSize`, or `""`. If unspecified or set to `""`, the default retention type is `RetentionSize` with 15 maximum backups. Specifying a retention type overrides the default value.
+
+[TIP]
+====
+Use dynamically-provisioned storage to keep the created etcd backup data in a safe, external location if possible. If dynamically-provisioned storage is not available, consider storing the backup data on an NFS share to make backup recovery more accessible.
+====
+
+:FeatureName: Automating etcd backups
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `FeatureGate` custom resource (CR) file named `etcd-enable-automated-backups.yaml` with the following contents:
++
+[IMPORTANT]
+====
+QUESTION: Are there any side effects of the `CustomNoUpgrade` feature set to warn the user about? 
```

Comment:
Found the answer to this one myself in the API docs:

"Turning this feature set on IS NOT SUPPORTED, CANNOT BE UNDONE, and PREVENTS UPGRADES."

Seems like we should probably mention this. ;)